### DISCUSSION TITLE
How to download dictionaries to Zettlr?

### DISCUSSION THREAD

**Discussion by yael-84** [Zettlr/Zettlr#2542] [General]
Hi,

I'm very new to this and are trying to download a dictionary from github to use in zettlr but don't understand HOW to get the file downloaded?

Excuse my basic question.

Cheers
/Yael

**Comment by canpolat** [comment]
Documentation has a section on adding dictionaries: https://docs.zettlr.com/en/reference/spell-checking/#add-new-dictionaries

**Reply by yael-84** [reply]
@canpolat Thank you for your answer! But I still can't figure out how. This section says "Simply download one of the folders to your computer". Well, that's my problem, how do I get the file downloaded?

**Reply by canpolat** [reply]
I see what you mean. If you only need one directory, the simplest way could be to go into the dictionary folder in Github and download each file individually (right-click -> save file/link as...). Since typically there are 6-7 files, this shouldn't take longer than a minute or two. Once you have downloaded all the files (and copied to a folder with the correct name), you can follow the guide to import them into Zettlr.

In other words:

1. Create a directory with dictionary name in you PC. Use the name of the dictionary that is in the Github repo (for example, es-VE for Venezuelan Spanish)
2. Individually download files to your PC and copy them into the folder created in step 1.
3. Import dictionary into Zettlr following the guide.

Please let me know if you need further assistance.

**Comment by nathanlesage** [comment]
Quick note: The dictionaries are, in the way we're using them now, deprecated. At some point in the future we'll switch to simply the spellchecking API as provided by Electron. Works faster, more reliable, and makes my own life easier as well!
